docs/release_notes.md

```diff
@@ -1,5 +1,97 @@
+- Install msiExecCmd_bin/hello script as a template (#7994).
+- Replace stacktrace with user-friendly error message in `itree` (#8082).
+- Improve GenQuery2 parser's handling of whitespace sequences (#8182).
+- Add json_events.hpp header file to development package (#8200).
+
+### Removed
+
+- Remove CentOS 7 build from GitHub workflows (#7968).
+
+### Deprecated
+
+- Deprecate `ilocate` (#2524).
+- Deprecate `igetwild` (#2525).
+- Deprecate modification of user having rodsgroup type in GeneralAdmin API (#2978).     THIS ISSUE IS REFERENCED IN TWO SECTIONS. IT DOES COVER TWO THINGS.
+- Deprecate modification of ACL policy (#6843).
+- Deprecate GeneralRowInsert and GeneralRowPurge (#7608).
+- Deprecate update_deprecated_database_columns.py (#7835).
+- Deprecate `imeta qu` (#7959).
+- Deprecate `imeta` interactive mode (#7961).
+- Deprecate server monitoring microservices (#7977).
+- Deprecate DataObjLock and DataObjUnlock (#7979).
+- Deprecate unused global variables in resource manager implementation. (#8042).
+- Deprecate `fillGenQueryInpFromStrCond` (#8088).
+- Deprecate `forkAndExec` (#8109).
+
+### Fixed
+
+- Avoid SIGABRT by setting pointers to null after deallocation (#3581).
+- Return errors on one-sided encryption in parallel transfer (#4984).       THINK ABOUT THIS ONE. ISSUE IS OPEN, BUT HAS BEEN BUMPED TO 5.0 MILESTONE.
+- Use `ProcessType` to detect server vs pure client (#6684).
+- Rework logic in `_cllExecSqlNoResult` to return proper error codes (#7440, #7599).
+- Fix flex warning for GenQuery2 lexer rules (#7685).
+- Validate and reject invalid zone names (#7722).
+- Return non-zero exit code from `iadmin` on failure (#7734).
+- Remove undefined reference to variable in exception (#7752).
+- Ignore SIGPIPE so agents do not terminate immediately (#7933, #7934).
+- Change if-statement so ticket use-count is only updated on first pass (#7967).
+- Write client and proxy user info to `ips` data file in correct order (#8001).
+- Modify JSON schema file to enforce required attributes of `controlled_user_connection_list` (#8017).
+- Use `std::tolower` with `std::transform` correctly (#8045).
+- Fix error handling logic for heartbeat operation (#8050).
+- Return error on invalid GenQuery1 aggregate function (#8080).
+- Return error on nonexistent target for write functions in iRODS Rule Language (#8095).
+- Clean up memory in `rsDataObjChksum`, various endpoints, and iCommands (#8106).
+- Correct narrowing of floating point values (#8110).
+- Remove constructor from `Cache` data type in iRODS Rule Language to allow for `memset` (#8111).
+- Avoid segfault in `rcDisconnect` (#8120).
+- Fix libstdc++ linker error for `ienv`, `ierror`, and `ipwd` on EL8 (#8122).
+- Fix GenQuery2 column mappings for user zone (#8134).
+- Use correct GenQuery2 table aliases for permissions (#8135).
+- Fix `nullptr` to function argument and unsigned overflow (#8153).        THIS IS PART OF THE ONGOING MILESTONE.
```

Comment:
This is likely okay since it's no different that housekeeping issue numbers landing in the release notes.

Reply:
Done.